src/Playwright/Core/Tracing.cs

```diff
@@ -58,12 +67,12 @@ public async Task StopChunkAsync(TracingStopChunkOptions options = null)
         public async Task StopAsync(TracingStopOptions options = default)
         {
             await StopChunkAsync(new() { Path = options?.Path }).ConfigureAwait(false);
-            await _context.Channel.TracingStopAsync().ConfigureAwait(false);
+            await _channel.TracingStopAsync().ConfigureAwait(false);
         }
 
         private async Task DoStopChunkAsync(string filePath)
         {
-            bool isLocal = _context.Channel.Connection.IsRemote;
+            bool isLocal = _channel.Connection.IsRemote;
```

Comment:
Unrelated to this PR:
```suggestion
            bool isLocal = !_channel.Connection.IsRemote;
```
https://github.com/microsoft/playwright/blob/b58b004f0f397a7f1516d0b4e3da2a6937b0375a/packages/playwright-core/src/client/tracing.ts#L57